features/dd-sdk-android-trace-api/src/main/kotlin/com/datadog/android/trace/api/span/DatadogSpanBuilder.kt

```diff
@@ -34,8 +33,6 @@ interface DatadogSpanBuilder {
 
     /**
      * Adds a tag to the span with the specified key and a nullable double value.
-     * Tags are used to add contextual information to spans, which can later be
```

Comment:
why we are deleting this? not useful?

Reply:
Seems like being lost during one of squash/rebase procedure